integrations/qdrant/tests/test_document_store.py

```diff
@@ -1,25 +1,41 @@
 from typing import List
 
+import numpy as np
 import pytest
 from haystack import Document
+from haystack.dataclasses import SparseEmbedding
 from haystack.document_stores.errors import DuplicateDocumentError
 from haystack.document_stores.types import DuplicatePolicy
 from haystack.testing.document_store import (
     CountDocumentsTest,
     DeleteDocumentsTest,
     WriteDocumentsTest,
+    _random_embeddings,
 )
-from haystack_integrations.document_stores.qdrant import QdrantDocumentStore
+from haystack_integrations.document_stores.qdrant.document_store import QdrantDocumentStore, QdrantStoreError
 
 
-class TestQdrantStoreBaseTests(CountDocumentsTest, WriteDocumentsTest, DeleteDocumentsTest):
+def _generate_mocked_sparse_embedding(n):
```

Comment:
let's move this to `conftest.py` and make it a fixture, we shouldn't import from test modules